### DISCUSSION TITLE
Objects not showing up in main search after direct database import

### DISCUSSION THREAD

**Discussion by Azmodeszer** [netbox-community/netbox#14286] [Q&A] [answered]
Hello,

I added a large number of IP addresses to a few existing interfaces directly in the database (by setting the appropriate ID value for the assigned_object_id column). Everything worked fine... except for the IP objects now not showing up in the main search (they do show up in the quick search under IPAM/IP addresses, however). Is there some command I need to run to index the new objects for the search or something?

Yes, I realize that this was not the recommended way to go about it, and in hindsight there is no reason not to use the bulk import function of the webGUI. Lesson learned.

Thanks.

**Comment by candlerb** [comment, marked as answer]
Look in `/opt/netbox/upgrade.sh`:

```
# Rebuild the search cache (lazily)
COMMAND="python3 netbox/manage.py reindex --lazy"
echo "Rebuilding search cache ($COMMAND)..."
```

You may or may not need to remove `--lazy`

```
        parser.add_argument(
            '--lazy',
            action='store_true',
            help="For each model, reindex objects only if no cache entries already exist"
        )
```

**Reply by Azmodeszer** [reply]
ah, I've tried it with the --lazy flag before, but that didn't help. Ran it without now and all good, thanks.
